modules/distribution/src/sso/views/register_new_user.jag

```diff
@@ -0,0 +1,200 @@
+} else {
+    signInUrl = configs.urls.store + '/login?ignoreReferer=true';
+}
+
+var recoveryModule = require('account-management').recovery;
+var challengeSet = recoveryModule.getChallengeQuestionSet();
+var isPasswordRecoveryEnabled = recoveryModule.isPasswordRecoveryEnabled();
+var themeModule = require('/modules/theme.js');
+%>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Sign in or Register | <% include(themeModule.getThemedUrl('includes/title.jag')); %></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <% include(themeModule.getThemedUrl('includes/header_file_imports.jag')); %>
+</head>
+<body>
+<div class="container-fluid">
+        <% include(themeModule.getThemedUrl('includes/header.jag')); %>
+        <div class="row">
+                <!-- content -->
+                <div class="col-xs-12 col-sm-10 col-md-8 col-lg-5 col-centered wr-login">
+                    <form id="registerForm">
+                        <h2 class="wr-title uppercase blue-bg padding-double white boarder-bottom-blue margin-none">Create An Account</h2>
+                        <div class="clearfix"></div>
+                        <div class="boarder-all ">
+                            <div class="padding-double font-large">Enter all text fields to complete registration </div>
+                            <!-- validation -->
+                            <div class="padding-double">
+                                <div id="regFormError" class="alert alert-error" style="display:none"></div>
+                                <div id="regFormSuc" class="alert alert-success" style="display:none"></div>
+
+                                    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
+                                        <label>Username</label>
+                                        <input id="reg-username" name="reg-username" type="text" class="form-control required usrName usrNameLength">
+                                    </div>
+                                    <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 form-group">
+                                        <label>Password</label>
+                                        <input id="reg-password" name="reg-password" type="password" class="form-control required passwordLength">
+                                    </div>
+                                    <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 form-group">
+                                        <label>Confirm password</label>
+                                        <input id="reg-password2" name="password2" type="password" class="form-control required" equalTo="#reg-password">
+                                    </div>
+
+                                <%
+                                    var claimsModule = require('account-management');
```

Comment:
Be consistent on using quotes for string definitions. use double quotes as the standard.